neard/src/cli.rs

```diff
@@ -123,27 +123,28 @@ pub(super) enum NeardSubCommand {
     /// DEPRECATED: this command has been renamed to 'localnet' and will be removed in a future
     /// release.
     // TODO(#4372): Deprecated since 1.24.  Delete it in a couple of releases in 2022.
-    #[clap(name = "testnet")]
+    #[clap(name = "testnet", hide(true))]
     Testnet(LocalnetCmd),
     /// (unsafe) Remove the entire NEAR home directory (which includes the
     /// configuration, genesis files, private keys and data).  This effectively
     /// removes all information about the network.
-    #[clap(name = "unsafe_reset_all")]
+    #[clap(name = "unsafe_reset_all", hide(true))]
     UnsafeResetAll,
     /// (unsafe) Remove all the data, effectively resetting node to the genesis state (keeps genesis and
     /// config).
-    #[clap(name = "unsafe_reset_data")]
+    #[clap(name = "unsafe_reset_data", hide(true))]
     UnsafeResetData,
     /// View DB state.
-    #[clap(name = "view_state")]
+    #[clap(subcommand, name = "view_state")]
```

Comment:
Why does this one get `subcommand` but others don’t?

Reply:
correct me if i'm wrong but from my understand it seems like only state viewer is a _true_ subcommand, because from within state viewer you can call more commands from `StateViewerSubCommand` enum, versus for example `RecompressStorageSubCommand` where you only provide the  `--output-dir`. Perhaps it's more of a discussion of the naming/semantics(?)

Reply:
> from my understand it seems like only state viewer is a true subcommand, because from within state viewer you can call more commands

Yes, that’s correct. If that’s the distinction for what is a ‘subcommand’ that that answers my question.